### ISSUE TITLE
Crash: Unrecoverable crash when breaking Woot mob farm connected to Powercell

### ISSUE THREAD

**Issue by eslachance** [McJty/RFTools#1080]
#### Issue description:
Breaking the Mob Controller on Woot multiblock structure, if powered by an RFTools Powercell, crashes single player worlds and servers alike, and crashes again as soon as the chunk loads.

*Note*: I'm reporting this on RFTools because the error specifically talks about power: 
```
java.lang.NullPointerException: Ticking block entity
	at mcjty.lib.varia.EnergyTools.receiveEnergy(EnergyTools.java:120)
[...]
-- Head --
Thread: Server thread
Stacktrace:
	at mcjty.lib.varia.EnergyTools.receiveEnergy(EnergyTools.java:120)
	at mcjty.rftools.blocks.powercell.PowerCellTileEntity.sendOutEnergy(PowerCellTileEntity.java:316)
	at mcjty.rftools.blocks.powercell.PowerCellTileEntity.func_73660_a(PowerCellTileEntity.java:260)

-- Block entity being ticked --
Details:
	Name: rftools_powercell // mcjty.rftools.blocks.powercell.PowerCellNormalTileEntity
	Block type: ID #2807 (tile.rftools.powercell // mcjty.rftools.blocks.powercell.PowerCellBlock)
	Block data value: 0 / 0x0 / 0b0000
```
#### What happens:
Setup a Woot multiblock structure. Attach Powercell. Break the Mob Controller. 

#### What you expected to happen:
Mob Controller drops, powercell stops attempting to provide power.

#### Steps to reproduce:
1. Create Tier 1 Woot mob spawner. 
2. Attach at least one Extender + Remote controller underneath
3. Attach Chest and RFTools Powercell (creative or otherwise) to remote
4. Break Mob Controller
5. *Note*: The chunk can no longer be loaded as long as the powercell is present.

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 12.18.3.2254
- Woot: 1.2.5 
- RFTools: 5.87

Does this issue occur on a server? Yes, but can also reproduce on single player.
Modpack Used: FTB Beyond (1.0.1)

Full Crash Log: http://pastebin.com/fYcNGtKZ

**Comment by eslachance** [comment]
Note that there is a related ticket on Woot: https://github.com/Ipsis/Woot/issues/141 
I think the issue probably just isn't fixed in 1.10.2. Closing this because... well, not RFTool's fault if I read the comments properly!

**Comment by Ipsis** [comment]
McJty please ignore. I never released the fix for 1.10, which I will be doing today :(

**Comment by McJty** [comment]
Ok, I will try very hard to ignore this...

Thanks :-)
